webrtc-stats.html

```diff
@@ -1971,12 +1972,21 @@
               <dd>
                 <p>
                   It is the total time each audio sample or video frame takes from the time it is
-                  received to the time it is rendered. The delay is measured from the time the
-                  first packet belonging to an audio/video frame enters the jitter buffer to the
-                  time the complete frame is sent for rendering after decoding. The average jitter
-                  buffer delay can be calculated by dividing the <code>jitterBufferDelay</code>
-                  with the <code>framesDecoded</code> (for video) or
-                  <code>totalSamplesReceived</code> (for audio).
+                  received and enters the jitter buffer to the time it exits the jitter buffer. This
+                  increases upon samples/frames exiting, having completed their time in the buffer
+                  (incrementing <code>jitterBufferOutput</code>). The average jitter buffer delay
+                  can be calculated by dividing the <code>jitterBufferDelay</code> with the
+                  <code>jitterBufferOutput</code>.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>jitterBufferOutput</code></dfn> of type <span class=
```

Comment:
The name doesn't sound right to me, but I can't think of a better name, because we want it to cover both samples and frames, and have all the jitterbuffer items start with "jitterBuffer".
jitterBufferEmitted? jitterBufferEmittedCount? jitterBufferEmittedSamplesCount? (too long!)

Reply:
Maybe we should have separate counters for frames (video) and samples (audio).

Reply:
I went with jitterBufferEmittedCount.